network-formula/network/wicked/interfaces.sls

```diff
@@ -0,0 +1,143 @@
+{#-
+Salt state file for managing network interfaces using Wicked
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'network/wicked/map.jinja' import base, base_backup, ifcfg_defaults, interfaces, script -%}
+{%- set ifcfg_data = {} %}
+{%- set startmode_ifcfg = {'auto': [], 'off': []} %}
+
+wicked_backup_directory:
+  file.directory:
+    - name: {{ base_backup }}
+    - mode: '0750'
+
+wicked_script:
+  file.managed:
+    - name: {{ script }}
+    - source: salt://{{ slspath }}/files{{ script }}
+    - mode: '0770'
```

Comment:
0750 (= not group-writeable) would be better

Reply:
Good catch, thanks!